pull-request/src/person.js

```diff
@@ -11,5 +11,7 @@ Person.prototype.fullname = function() {
 }
 
 Person.prototype.wishHappyBirthday = function() {
+    console.log("Wishing happy birthday");
     return "Happy birthday " + this.firstName;
+    this.age++;
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Remove this code after the "return" statement. ([javascript:UnreachableCode](http://localhost:9075/coding_rules#rule_key=javascript%3AUnreachableCode))

[See it in SonarQube](http://localhost:9075/project/issues?id=training%3Apull-request&pullRequest=42&issues=AWgYy5hfUHOAr7vtaoKI&open=AWgYy5hfUHOAr7vtaoKI)

[ISSUE_KEY]: AWgYy5hfUHOAr7vtaoKI